phalcon/db/adapter/pdo.zep

```diff
@@ -85,9 +85,12 @@ abstract class Pdo extends Adapter
 	 */
 	public function connect(descriptor = null)
 	{
-		var username, password, dsnParts, dsnAttributes,
+		var eventsManager,
+			username, password, dsnParts, dsnAttributes,
 			persistent, options, key, value;
 
+		let eventsManager = <ManagerInterface> this->_eventsManager;
```

Comment:
It could even log a message or throw an exception to let people know what to change.  A nice deprecation notice doesn't work because then sometimes events are available and sometimes not.